webapp/commonComponents/errorBadge.js

```diff
@@ -34,7 +38,12 @@ const ErrorBadge = ({ validation, showLabel, label }) => {
 ErrorBadge.defaultProps = {
   validation: null,
   showLabel: true,
-  label: 'INVALID'
+  label: 'INVALID',
+  /**
+   * error message to show if validation is invalid
+   * (if not specified, field validation errors will be shown)
+   */
+  tooltipErrorMessage: null
 }
```

Comment:
can you just put one line comment in this case? e.g.
``` //error message to show if validation is invalid - if not specified, field validation errors will be shown```

Reply:
ok